app/controllers/home_controller.rb

```diff
@@ -4,17 +4,26 @@ def index
     home_path_for_user
   end
 
-  # FIX: Temporary controller, remove me when no longer required
   def quality
     @scores = QualityScore.all.order(median: :desc, highest: :desc)
   end
 
+  def dashboard
```

Comment:
Any particular reason why the dashboard is being managed by the HomeController?

Reply:
I agree, it's weird. This is ripe for a refactor.

I'd rather have a `Pages` controller for all our static pages, like we have in FIND, but this is what we have at the moment.

I thought about refactoring this too, but found it is out of the scope of this PR.